src/osd/PGBackend.cc

```diff
@@ -1041,7 +1053,7 @@ void PGBackend::be_compare_scrubmaps(
       }
       if (auth_object.digest_present && auth_object.omap_digest_present &&
 	  cct->_conf->osd_debug_scrub_chance_rewrite_digest &&
-	  (((unsigned)rand() % 100) >
+	  ((return_rand(errorstream) % 100) >
```

Comment:
Using file I/O for random nubmer generation is by definition slow, and in that case it might be actually *desireable* to have repeatable sequence of pseudo random numbers. This coverity warning is without merit here.